extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -10,7 +10,7 @@ function ($, mw, loader, nirvana, tracker, labeling) {
 	'use strict';
 
 	var $classificationForm,
-		$preselectedType,
+		$preselectedType = $(),
```

Comment:
I think we don't need to init this value as empty. Probably this fix is enough https://github.com/Wikia/app/pull/8976/files#diff-d5bde9afc6a40ff66cf72bf44a3fc4b2R82